### DISCUSSION TITLE
How to use the bookmark exporting function?

### DISCUSSION THREAD

**Discussion by matt873** [JingMatrix/ChromeXt#62] [Q&A] [answered]
I searched for the user scripts option in Chrome after enabling the module but didn't find that. So how does this app work? How can I import/insert user scripts?

P.S. I don't know much about user scripts and just wanted to try this app to export bookmarks but didn't find any tutorials on youtube.

**Comment by JingMatrix** [comment]
In your browser, you can install UserScripts from popular websites such as [GreasyFork](https://greasyfork.org/): just click the install buttons there after that you find an interesting script.
There is a [front-end websites](https://jingmatrix.github.io/ChromeXt/) where you can manage your installed scripts.

Importing UserScripts is not implemented yet.

**Comment by matt873** [comment]
Thank you. I want to use it for just exporting bookmarks. How can I do that as I didn't find any userscripts for that in GreasyFork?

**Comment by JingMatrix** [comment]
Please read the README, it is explained there.
You need to enable the Developers options menu.

**Comment by matt873** [comment]
I can access the developer option from menu on the ChromeXt manager page. There are two options there, ChromeDev Tools and Inspect. What can I do from there to export bookmarks. 
P.S. I don't know much about user scripts and just wanted to try this app to export bookmarks but didn't find any tutorials on youtube. So your help will be greatly appreciated!

**Comment by JingMatrix** [comment, marked as answer]
What you opened was `Developer Tools` menu, not `Developer options` menu.

Please read this section: https://github.com/JingMatrix/ChromeXt#solution-of-system-gesture-conflicts
1. Step one: Tap seven times on the Chrome version from the Chrome settings, you will see the Developer options menu.
2. Step two: Bookmarks can be exported in HTML format through the Developer options menu.

**Comment by JingMatrix** [comment]
Here is a [blog](https://www.askvg.com/tip-enable-hidden-secret-developer-options-menu-in-chrome-settings-in-mobile-phones/) with a video telling you how to find the `Developer options` menu.
It should be helpful for you.

**Comment by matt873** [comment]
Thank you so much for your kind help! I was able to successfully export bookmarks with a few clicks. Thank you so much again!
